### DISCUSSION TITLE
Help optimize 2x Xeons , 32gb ram, SAS drives

### DISCUSSION THREAD

**Discussion by ankomen2** [swar/Swar-Chia-Plot-Manager#744] [Q&A]
I have 2x x5670 , 12 core , 24 threads,32gb, 4 sas drives 300gb each. I plan on running 4 threads per plot. Now with GUI, I get 1 plot per 20 hours. Any suggestion and settings to put in the config file. Thanks :)

**Comment by Saviola8x** [comment]
You should run in linux env with better performance. use 4-8 thread , 6-8000 MB Buffer per job, and should use sas 6/12Gbps not recommended sas 3Gbps with worst I/O, final_directory should be SATA/SAS 6TB++. I completed plot around 11-15 hours depend on disk performance.

**Reply by ankomen2** [reply]
In buffer you mean ram right. I use sas 6 gbps. In crystal disk I measure 350 mb/s. I should be able to hit 11-15 hour mark

**Reply by Saviola8x** [reply]
> In buffer you mean ram right. I use sas 6 gbps. In crystal disk I measure 350 mb/s. I should be able to hit 11-15 hour mark

Yep, it's memory_buffer , in default config is 4000 but i has been checked plotting log and see sometime memory exhausted. You should set higher. And windows is always slower than linux as 10-15% (chia docs was metioned).Linux CLI maybe seem hard with newbie but installation is cleared so you can do it easy.

**Reply by ankomen2** [reply]
Will I be able to transfer to linux because I am almost synced :/

**Reply by Saviola8x** [reply]
No problem, NTFS disk can be mounted in linux (ex. ubuntu with ntfs-3g). You can google in few minute to do it. Please sure that you shutdown windows gracefully to avoid mounting problem.

**Reply by ankomen2** [reply]
Yea but my operating system takes one of my SAS 300gb disk (don't ask me why :D)I want to delete it. So I just copy the chia blockchain folders like  https://thechiafarmer.com/2021/04/26/how-to-move-a-chia-farm-to-another-pc/ but I don't know where to put it in the linux distro. One last question can I run linux of of 8/16gb usb live and still be able to plot. Thanks btw for your answers <3

**Reply by Saviola8x** [reply]
Some linux os can be work on usb us live image, but it's not recommended for production, but you can try :
- run os (ubuntu 20.04) on usb and use 4 x sas 300gb for temporary and dest directory as NAS/external disk. But this can be slow down your plotting time, at least in transfer from temp to destination disk (as my plot machine, transfer time from temp sas 300gb  to local destination disk (SATA) is around 15min), and I dont run os in usb :D 
- run os + temp folder in 1 of disk, your os maybe slow sometime but not much.

About transfer, you only need save 24 words, and use command "chia keys add" (after install chia-blockchain) - put 24 words and your keys will be imported.

**Reply by ankomen2** [reply]
I guess linux is better one command and everything is there :D. Windows is using 40-50 gb space and I can't plot on that drive stupid me. So I will not be syncing form the start again with chia keys add. That's amazing. The GUI is unfinished I guess. Thanks again I will do that :)
